assets/js/components/notifications/BannerNotifications.js

```diff
@@ -70,6 +73,14 @@ export default function BannerNotifications() {
 	const hasGTMScope = useSelect( ( select ) =>
 		select( CORE_USER ).hasScope( GTM_SCOPE )
 	);
+	const isAnalyticsModuleOwner = useSelect( ( select ) =>
+		// Here we check for a capability that only the module owner would have
+		// since the ownerID setting won't be available in a view-only context.
+		select( CORE_USER ).hasCapability(
```

Comment:
Hmm - it seems to me we would never want to show these notifications on the view-only Dashboard - after all they're prompting the user to do things that are not applicable in a view-only context, and I'm not sure the current user could actually be the owner if they're not connected anyway.

With that in mind, it seems it would be more straightforward and correct here to return early if we're currently in view-only mode, otherwise check the owner ID as usual. What do you think?